packages/arb-bridge-eth/contracts/arch/OneStepProof.sol

```diff
@@ -1562,19 +1562,19 @@ library OneStepProof {
             }
         }
 
-        require(_data.beforeHash == startMachine.hash(), "Proof had non matching start state");
-        require(_data.afterHash == endMachine.hash(), "Proof had non matching end state");
-
-        // require(
-        //     _data.beforeHash == startMachine.hash(),
-        //     string(abi.encodePacked("Proof had non matching start state: ", startMachine.toString(),
-        //     " beforeHash = ", DebugPrint.bytes32string(_data.beforeHash), "\nstartMachine = ", DebugPrint.bytes32string(startMachine.hash())))
-        // );
-        // require(
-        //     _data.afterHash == endMachine.hash(),
-        //     string(abi.encodePacked("Proof had non matching end state: ", endMachine.toString(),
-        //     " afterHash = ", DebugPrint.bytes32string(_data.afterHash), "\nendMachine = ", DebugPrint.bytes32string(endMachine.hash())))
-        // );
+        // require(_data.beforeHash == startMachine.hash(), "Proof had non matching start state");
+        // require(_data.afterHash == endMachine.hash(), "Proof had non matching end state");
+
+        require(
+            _data.beforeHash == startMachine.hash(),
+            string(abi.encodePacked("Proof had non matching start state: ", startMachine.toString(),
+            " beforeHash = ", DebugPrint.bytes32string(_data.beforeHash), "\nstartMachine = ", DebugPrint.bytes32string(startMachine.hash())))
+        );
+        require(
+            _data.afterHash == endMachine.hash(),
+            string(abi.encodePacked("Proof had non matching end state: ", endMachine.toString(),
+            " afterHash = ", DebugPrint.bytes32string(_data.afterHash), "\nendMachine = ", DebugPrint.bytes32string(endMachine.hash())))
+        );
```

Comment:
This should have the simple requires uncommented and the fancy/expesive ones comments out